src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+/*eslint-disable */
+class Calculator extends Component {
```

Comment:
- [ ] Kindly enable `eslint`. You can disable the rule for the `prefer-stateless-function` but do not disable all eslint rules.